modules/post/windows/gather/forensics/memorydump.rb

```diff
@@ -0,0 +1,230 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+#
+# This Module has an embred Belkasoft RAM Capturer version of 2018-08
+# Available at https://belkasoft.com/ram-capturer
+##
+
+#
+# Forensic memory dump using Belkasoft RAM Capturer
+#
+# Helvio Junior (M4v3r1cK) m4v3r1ck.hjr [at] gmail.com
+#    https://www.helviojunior.com.br
+#
+# Other modules repository: https://github.com/helviojunior/metasploit_modules
+#
+# Standard Library
+#
+require 'tmpdir'
+
+
+class MetasploitModule < Msf::Post
+  include Msf::Post::File
+  include Msf::Post::Windows::UserProfiles
+
+  def initialize(info={})
+    super( update_info( info,
+      'Name'          => 'Windows Gather Memory Dump Imaging',
+      'Description'   => %q{This module will perform a memory dump of remote machine},
```

Comment:
If I understand correctly, this module uploads a copy of Belkasoft RAM Capturer, which needs to be present somewhere on the operator's file system. This should be explained here.